Specs/DateTime/German/MergedExtractor.json

```diff
@@ -86,26 +86,26 @@
     ]
   },
   {
-    "Input": "06/06 12:15",
+    "Input": "6.6. 12:15",
```

Comment:
Same

Reply:
I would not use the / form as this will case ambiguity. Is 06/06 June 6th or June 2006? In English it could be both, in German it would be rather the Month/Year. What do you think? If we want to test for 06/06 in German as well we also need to add specs to the MergedParser that returns three results day/month, month/day and month/year.